translator.py

```diff
@@ -0,0 +1,129 @@
+
+    #Code for Short trades:
+
+    elif Trade_Type == "Short":
+        Shares = (abs(50 / (Stop - Trigger)))
+        target_below = "%.2f" %(Trigger - (Stop - Trigger))
+        one_one_partial = input("Did you reach your 1:1 target of " + str(target_below) + "?")
+
+        # first "if" checks if any of the trade succeeded
+
+        if one_one_partial == "no":
+            Executed_stop = float(input("When did Stop execute?"))
+            loss = abs(Executed_stop - Executed_trigger) * Shares
+            print("You lost " + str(loss) + "$ on all trades")
+        elif one_one_partial == "yes":
+            one_one_partial_profit = "%.2f" % ((Executed_trigger - (Trigger - (Stop - Trigger))) * 0.75 * Shares)
+            one_one_full_profit = "%.2f" % ((Executed_trigger - (Trigger - (Stop - Trigger))) * Shares)
+            remainder_one_one = float(input("Where did you cover remainder of 1:1 trade: "))
+            remainder_one_one_profit = "%.2f" % ((Executed_trigger - remainder_one_one) * 0.25 * Shares)
+
+            # This "if" checks if a 2:1 type of trade succeeded
+            target_beloww = "%.2f" %((Trigger - 2 * (Stop - Trigger)))
+            two_one_partial = input("Did you reach your 2:1 target of " + str(target_beloww) + str("?"))
+            if two_one_partial == "no":
+                Executed_stop = float(input("When did Stop execute?"))
+                loss = "%.2f" % (abs(Executed_stop - Executed_trigger) * Shares)
+                print("Your 2:1 trades have lost: " + str(loss) + "$")
+                print("If you traded 1:1 100% you earned " + str(one_one_full_profit) + "$")
+                print("If you traded 1:1 75% 25% you earned " + str(one_one_partial_profit) + "$ + " + str(remainder_one_one_profit) + "$")
+            elif two_one_partial == "yes":
+                two_one_partial_profit = "%.2f" % ((Executed_trigger - (Trigger - (Stop - Trigger) * 2)) * 0.5 * Shares)
+                target_belowww = "%.2f" % (Trigger - 4 * (Stop - Trigger))
+                four_one_partial = input("Did you reach your 4:1 target of " + str(target_belowww) + str("?"))
+                if four_one_partial == "no":
+
+                    # if trade didn't make it to 1:4 it means all trades have been covered at this stage. I will print results.
+
+                    print("All trades concluded:")
+                    print("If you traded 1:1 100% you made " + str(one_one_full_profit) + "$")
+                    print("If you traded 1:1 75% 25% you made " + str(one_one_partial_profit) + "$ + " + str(remainder_one_one_profit) + "$")
+                    print("If you traded 2:1 variation you made " + str(two_one_partial_profit) + "$")
+
+                elif four_one_partial == "yes":
+
+                    # This "if" is for fully successfull 2:1 trades. I will calculate and print results.
+
```

Comment:
Document what you are doing exactly in the following 4 lines.

Reply:
Left spaces for organization. 
The "#" is to explain what's the purpose of the code 'paragraph'.
why are we writing in english?